File: prvy_program/prvy/src/prvy/Prvy.java
package prvy;

import java.util.Scanner;


public class Prvy {

    
    public static void main(String[] args) {
        Scanner reader = new Scanner(System.in);
        System.out.println("Zaádaj prve číslo : ");
        float prve = reader.nextFloat();
        
        System.out.println("Zaádaj druhe číslo : ");
        float druhe = reader.nextFloat();
        
        float vysledok = prve/ druhe;
        
        System.out.println("Vysledok je : " + vysledok);
        
    }
    
}

File: prvy_program/prvy.c
#include <stdio.h>

int main(){
	int prve, druhe;
	float vysledok = 0;
	
	printf("Zadaj prve cislo: ");
	scanf("%d" ,  &prve);
	printf("Zadaj druhe cislo: ");
	scanf("%d" ,  &druhe);
	
	vysledok = prve / druhe;
	
	printf("Vysledok je: %.2f", vysledok);
}
